src/software/thunderscope/robot_communication.py

```diff
@@ -37,18 +38,20 @@ def __init__(
         :param current_mode: one of RobotCommunicationMode, indicates which mode is currently running
         :param multicast_channel: The multicast channel to use
         :param interface: The interface to use
+        :param disable_estop: whether to disable estop checks (ONLY FOR TESTING LOCALLY)
         :param estop_path: The path to the estop
         :param estop_baudrate: The baudrate of the estop
 
         """
         self.sequence_number = 0
         self.last_time = time.time()
         self.current_proto_unix_io = current_proto_unix_io
+        self.current_mode = current_mode
```

Comment:
is this addition intentional?

Reply:
This looks to be an existing variable which was moved from lower down.